float64 gray_water_liters
---
bool success
string message
